### ISSUE TITLE
Legend keyboard navigation does not work in non-Chrome browsers

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#6511]
The legend items are not navigable using the a11y module in (most) non-Chrome browsers (e.g. Safari).

This is due to browsers not implementing programmatic focusing for SVG elements.

It is possible that we could work around this by letting the browser handle the tabindex logic, using `tabindex=0`.

**Comment by oysteinmoseng** [comment]
In IE11, tabbing goes through the series paths instead, apparently.

**Comment by PaulDalek** [comment]
+1

**Comment by oysteinmoseng** [comment]
This issue is resolved in the upcoming v7.1 release
